main.go

```diff
@@ -26,6 +26,14 @@ var (
 )
 
 func main() {
```

Comment:
It would be better to use Go's flag package for maintainability.

Reply:
Done.